tool/codegen/Dockerfile

```diff
@@ -41,6 +41,17 @@ RUN go install github.com/envoyproxy/protoc-gen-validate@v${PROTOC_GEN_VALIDATE_
   && rm protoc-gen-validate.tar.gz \
   && mv /go/src/github.com/envoyproxy/protoc-gen-validate-${PROTOC_GEN_VALIDATE_VER} /go/src/github.com/envoyproxy/protoc-gen-validate
 
+# protoc-gen-js
+# This is a workaround to use it  https://github.com/protocolbuffers/protobuf-javascript/issues/127#issuecomment-1361047597
+RUN for target in x86_64 aarch_64; do \
+  mkdir /protoc-gen-js-${target} && cd /protoc-gen-js-${target} \
+  && wget https://github.com/protocolbuffers/protobuf-javascript/releases/download/v${PROTOC_GEN_JS_VER}/protobuf-javascript-${PROTOC_GEN_JS_VER}-linux-${target}.tar.gz \
+  && tar xvfz protobuf-javascript-${PROTOC_GEN_JS_VER}-linux-${target}.tar.gz \
+  && chmod +x bin/protoc-gen-js \
+  && rm -rf protobuf-javascript-${PROTOC_GEN_JS_VER}-linux-${target}.tar.gz; \
+  done && \
+  mv /protoc-gen-js-aarch_64/ /protoc-gen-js-aarch64/ 
```

Comment:
📝 Get the binary not only for x86_64 but also aarch_64 to check it on the local machine.